docs/guides/admin/docs/css/extra.css

```diff
@@ -160,13 +160,6 @@ a.header-link:hover {
     opacity: 1;
 }
 
-details {
-  border-left: 2px solid silver;
-  padding: 5px;
-  margin-bottom: 10px;
-  margin-left: 10px;
-}
-
```

Comment:
Why remove the styling for `details`? That's used in other places as well.

Reply:
I figured this was a quick and easy way to make the changelog look better. I also looked at the other places `details` is used in, and found this acceptable. But I don't have a strong opinion on this, if you want this reverted I'm not against it.

Reply:
Pictures for comparison (the second ones are with the removed style):
![Bildschirmfoto vom 2023-10-25 09-34-48](https://github.com/opencast/opencast/assets/14070005/d77d8ae3-e1a2-4afb-a89c-d24ffd5a41c6)
![Bildschirmfoto vom 2023-10-25 09-36-06](https://github.com/opencast/opencast/assets/14070005/7e2e56d2-5722-4672-85aa-16820bf50eec)
![Bildschirmfoto vom 2023-10-25 09-37-23](https://github.com/opencast/opencast/assets/14070005/cdeddbbe-d8a5-47ea-8c6d-8e0e8f4755d1)
![Bildschirmfoto vom 2023-10-25 09-37-08](https://github.com/opencast/opencast/assets/14070005/681fec66-e3ca-4185-bb53-ee03a9ba25a4)